test_unstructured/partition/test_api.py

```diff
@@ -140,13 +136,14 @@ def test_partition_via_api_with_no_strategy():
     elements_hi_res = partition_via_api(filename=filename, strategy="hi_res", api_key=get_api_key())
 
     # confirm that hi_res strategy was not passed as default to partition by comparing outputs
-    # FIXME(crag): elements_hi_res[4].text is 'sacon oot barvard o', the fast output.
-    # should be 'Harvard University {melissadell,jacob carlson}@fas.harvard.edu' (as of writing)
+    # elements_hi_res[3].text =
+    #     'LayoutParser: A Uniﬁed Toolkit for Deep Learning Based Document Image Analysis'
+    # while elements_no_strategy[3].text = ']' (as of this writing)
     assert elements_no_strategy[4].text != elements_hi_res[4].text
 
 
-@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
-@pytest.mark.skipif(skip_not_on_main, reason="Skipping test run outside of main branch")
+# @pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
+# @pytest.mark.skipif(skip_not_on_main, reason="Skipping test run outside of main branch")
```

Comment:
i think we still want these skips, i.e. don't hit the live API frequently in tests

Reply:
Whoops. Thanks